careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
                   M1  > M2  > M3  > M4 > E1 > E2
 ```
 
+### Evaluation criteria
+
+- Knowledge Leadership
+  - Level of knowledge in a specific area
```

Comment:
We can be more specific about the _area_ here. Maybe we want to point out business/domain areas and differentiate from the technical skill sets below?